### ISSUE TITLE
Some decorations broken

### ISSUE THREAD

**Issue by R3flector** [grammyjs/nestjs#41]
@ChatType and @UpdateFilter not working

**Comment by KnorpelSenf** [comment]
Remember that when you say it “doesn't work”, we cannot login to your computer and figure out what's wrong. You need to explain what you are doing, what you are expecting to happen, and what is actually happening. If there's an error, share the full error message, too :)

**Comment by R3flector** [comment]
nest new myproject (with pnpm)
replace src folder from sample-echo-bot / change .env
pnpm add debug grammy @grammyjs/nestjs @grammyjs/types
pnpm start:dev


---
[10:34:00 PM] Starting compilation in watch mode...

src/app/bot/bot.update.ts:11:3 - error TS2305: Module '"@grammyjs/nestjs"' has no exported member 'ChatType'.

11   ChatType,
     ~~~~~~~~

src/app/bot/bot.update.ts:13:3 - error TS2305: Module '"@grammyjs/nestjs"' has no exported member 'UpdateFilter'.

13   UpdateFilter,
     ~~~~~~~~~~~~

[10:34:01 PM] Found 2 errors. Watching for file changes.
---

**Comment by morty6688** [comment]
well, it seems that no one is maintaining this project anymore...

**Comment by KnorpelSenf** [comment]
Yep, it's very hard to find people who want to do this. It seems like nobody uses nest because they're passionate about the technology, and everybody who uses it due to corporate reasons isn't allowed the resources to actually help out the projects they rely on. As a consequence, there's been very little activity here ever since this repo was created.
